test/framework/dump.go

```diff
@@ -173,16 +173,23 @@ func (f *CommonFramework) DumpLogsForPodsWithLabelsInNamespace(ctx context.Conte
 
 	var result error
 	for _, pod := range pods.Items {
-		if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name, &corev1.PodLogOptions{}); err != nil {
-			result = multierror.Append(result, err)
+		for _, container := range pod.Spec.InitContainers {
+			if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name, &corev1.PodLogOptions{Container: container.Name}); err != nil {
+				result = multierror.Append(result, fmt.Errorf("error reading logs from init container %q: %w", container.Name, err))
+			}
+		}
+		for _, container := range pod.Spec.Containers {
+			if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name, &corev1.PodLogOptions{Container: container.Name}); err != nil {
+				result = multierror.Append(result, fmt.Errorf("error reading logs from container %q: %w", container.Name, err))
```

Comment:
Please also add the pod name to the error message. Container name alone is not very helpful alone.